tabs/mission_control.js

```diff
@@ -2942,7 +2950,8 @@ TABS.mission_control.isBitSet = function(bits, testBit) {
 
 TABS.mission_control.setBit = function(bits, bit, value) {
     if ((initParam3 & (1 << bit)) != 0) {
```

Comment:
What is the purpose of `initParam3 `? I tested this with just:

    `return value ? bits |= (1 << bit) : bits &= ~(1 << bit);`

and it seemed to work as expected.

Reply:
This is needed in the initial setup of the checkboxes. Without this test, when the checkboxes are set up (lines 1823 -1827). setting the correct value for the checkbox triggers the onChange event. Which then changes the value. This test stops that from happening.

Reply:
Got to admit I'm not seeing that. With `initParam3 `removed it doesn't seem to make any difference. The onChange event triggers but it just sets P3 again to the same correct setting. Or is this just to avoid unwanted trigger events which do seem to happen a lot with JS regardless.